components/prism-elixir.js

```diff
@@ -1,4 +1,7 @@
 Prism.languages.elixir = {
+	'doc': {
+		pattern: /@(?:doc|moduledoc)\s+("{1,3}|'{1,3})(?:\\[\s\S]|(?!\1)[^\\])+\1/
+	},
 	'comment': /#.*/m,
```

Comment:
Comment has to be greedy or else `# @doc "abc"` will not be matched correctly.

```suggestion
	'comment': {
		pattern: /#.*/m,
		greedy: true
	},
```